contrib/platform/src/com/sun/jna/platform/win32/Wevtapi.java

```diff
@@ -0,0 +1,702 @@
+/* Copyright (c) 2016 Minoru Sakamoto, All Rights Reserved
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ */
```

Comment:
Please adjust the license header. The project is dua-licensed as LGPL 2.1 or later and AL 2.0. The source files were missed when the relicensing happend, which will be corrected by:

https://github.com/java-native-access/jna/pull/711

The intended header would be:

      /* Copyright (c) 2016 Minoru Sakamoto, All Rights Reserved
       * 
       * The contents of this file is dual-licensed under 2 
       * alternative Open Source/Free licenses: LGPL 2.1 or later and 
       * Apache License 2.0. (starting with JNA version 4.0.0).
       * 
       * You can freely decide which license you want to apply to 
       * the project.
       * 
       * You may obtain a copy of the LGPL License at:
       * 
       * http://www.gnu.org/licenses/licenses.html
       * 
       * A copy is also included in the downloadable source code package
       * containing JNA, in file "LGPL2.1".
       * 
       * You may obtain a copy of the Apache License at:
       * 
       * http://www.apache.org/licenses/
       * 
       * A copy is also included in the downloadable source code package
       * containing JNA, in file "AL2.0".
       */